doc/crypto/api/ops/pake.rst

```diff
@@ -1490,7 +1490,7 @@ The key pair is derived from the initial SPAKE2+ password prior to starting the
 It is recommended to use a key-stretching derivation algorithm, for example PBKDF2.
 This process can take place immediately before the PAKE operation, or derived at some earlier point and stored by the participant.
 Alternatively, the Verifier can be provisioned with the `PSA_KEY_TYPE_SPAKE2P_PUBLIC_KEY()` for the protocol, by the Prover, or some other agent.
-:numref:`fig-spake2p-reg` illustrates some example SPAKE2+ key derivation flows.
+:numref:`fig-spake2p-reg` illustrates some example SPAKE2+ key-derivation flows.
 
 The resulting SPAKE2+ key-pair must be protected at least as well as the password.
```

Comment:
Conversely this should be “key pair”. From `git grep -Eon 'key-\w+ \w+' doc` (I haven't combed the “key-foo and/or/with”).

We also seem to be inconsistent about keystore vs key-store.

Reply:
I will review key-pair/key pair, public-key/public key and key-store/keystore. For the latter I am inclined to go with "keystore".